packages/medusa/src/api/routes/store/carts/create-line-item.ts

```diff
@@ -78,4 +79,8 @@ export class StorePostCartsCartLineItemsReq {
 
   @IsOptional()
   metadata?: Record<string, unknown> | undefined
+
+  @IsBoolean()
+  @IsOptional()
+  validate_sales_channels = false
```

Comment:
suggestion: Same as previous handler

Reply:
same - customers (i.e. store route) don't get to decide if sales channel is validated